rskj-core/src/test/java/org/ethereum/datasource/KeyValueDataSourceTest.java

```diff
@@ -126,6 +132,19 @@ public void updateBatchWithNulls() {
         keyValueDataSource.updateBatch(updatedValues, Collections.emptySet());
     }
 
+    @Test
+    public void testShouldValidateKindWithEmptyDbDirAndResetDbFalseSuccessfully() {
```

Comment:
These two new tests are run 10 times, as this is a parameterised test, without any change on its params. This extra job is not needed, so maybe it's worth moving them to another file.

Reply:
Done